packages/plugin-ext/src/plugin/type-converters.ts

```diff
@@ -169,9 +169,9 @@ export function fromRangeOrRangeWithMessage(ranges: theia.Range[] | theia.Decora
         });
     } else {
         return ranges.map((r): DecorationOptions =>
-            ({
-                range: fromRange(r)!
-            }));
+        ({
```

Comment:
This is not the formatting the typescript formatter creates. Let's stick to that unless there is a compelling reason.

Reply:
Do you have directions on how to set this up? VS Code formats like this.

Reply:
@paul-marechal it doesn't for me 🤷